lib/gooddata/rest/client.rb

```diff
@@ -197,6 +191,10 @@ def connect
       end
 
       def disconnect
+        if stats_on?
+          puts "API call statistics to server #{@connection.server}"
```

Comment:
why do we can use it in https://github.com/gooddata/gooddata-ruby/blob/develop/lib/gooddata/lcm/lcm2.rb and not here? Any reasons?